style.css

```diff
@@ -0,0 +1,360 @@
+  color: white;
+  background-color: white;
+  margin-top: -10px;
+  margin-bottom: 30px;
+}
+
+#hero p {
+  font-size: 1.6rem;
+  margin-top: -10px;
+}
+
+#form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+
+#email {
+  margin-top: -5px;
+  height: 2rem;
+  width: 300px;
+  text-align: center;
+  border: 2px solid #000;
+  border-radius: 4px;
+  font-size: 1.1rem;
+}
+
+#submit {
+  margin-top: 5px;
+  height: 2rem;
+  width: 150px;
+  transition: color 0.6s ease, background-color 0.6s ease;
+  background-color: var(--back-color);
+  color: #000;
+  border: 2px solid #000;
+  border-radius: 4px;
+  font-size: 1rem;
+  text-align: center;
+  margin-top: 10px;
+  font-weight: 700;
+}
+
+#submit:hover {
+  background-color: #000;
+  color: var(--back-color);
+}
+
+/* End Hero Section */
+
+/* Main Section */
+
+#features {
+  position: relative;
+  margin: 3rem 45rem 2rem 45rem;
+  z-index: -1000;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  height: 20rem;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 800px;
+}
+
+#icon1 {
+  height: 60px;
+  width: auto;
+  margin-top: 25px;
+}
+
+#icon2 {
+  height: 60px;
+  width: auto;
+  margin-top: 20px;
+}
+
+#icon3 {
+  height: 60px;
+  width: auto;
+  margin-top: 30px;
+}
+
+.info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.info p {
+  margin-top: -20px;
+}
+
+#features .fhr {
+  color: #000;
+  background-color: #000;
+  width: 800px;
+  height: 5px;
+  border-width: 0;
+}
+
+/* video section */
+.video {
+  margin-top: 3.125rem;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.video iframe {
+  max-width: 35rem;
+  width: 100%;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  height: 12.5rem;
+  margin-top: 3.125rem;
+}
+
+@media (max-width: 1200px) {
+  footer {
+    background-color: #f3f3f3;
+  }
+}
+
+.form form {
+  max-width: 17.1875rem;
+  width: 100%;
+  padding: 0.3125rem;
+}
+
+.form p {
+  margin-bottom: 1.25rem;
+  word-wrap: break-word;
+}
+
+form input[type='email'] {
+  outline: none;
+  padding: 0.75rem 0.625rem;
+  width: 100%;
+  border: 1px solid rgba(153, 153, 153, 0.7);
+  outline: none;
+  font-size: 1rem;
+  border-radius: 0.25rem;
+  box-shadow: 0 1px 2px 0 rgba(0, 0, 0, 0.03);
+  transition: all 0.4s ease-in-out;
+}
+
+.form input[type='submit'] {
+  border: none;
+  height: 2.5rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  border-radius: 0.3125rem;
+  font-size: 0.9375rem;
+  line-height: 1rem;
+  font-weight: 600;
+  transition: background-color 0.1s ease-in;
+  cursor: pointer;
+  margin-top: 1.25rem;
+}
+
+.form input[type='submit']:hover {
+  background: #f1c40f;
+}
+
+footer {
+  background: #f3f3f3;
+  padding: 0.9375rem 0rem;
```

Comment:
Unexpected unit

Reply:
- [ ] Please ensure to be clear in your communication. You should point out the particular unit that is unexpected, in this case, the "0rem" and state that to achieve the best CSS practice, when a value is 0, there is no need to include the unit. I suggest you explain that for the padding style, the developer should have, **padding: 0.9375rem 0;**